src/System.Linq.Parallel/tests/WithCancellationTests.cs

```diff
@@ -185,17 +185,15 @@ public static void CTT_NonSorting_SynchronousMergerEnumeratorDispose()
             Assert.NotNull(caughtException);
         }
 
-        
-
         /// <summary>
-        /// 
+        ///
```

Comment:
Yeah, I'll probably revert this file too, because I don't think I've touched it otherwise.